0_intro_to_jax_for_ml.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>Nit: missing ')', 'has' -&gt; 'have'</p><p> </p><p>I am not sure that I would use <code>pmap</code> to create these matrices. I think it might be more clear to reuse the previous matrices and focus on explaining <code>pmap</code> for the batched dot product?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='107'/>

Reply:
<p>Made spelling fixes. So using pmap makes it a <code>ShardedDeviceArray</code> , which is what you need to <code>pmap</code> effectively.   </p>